orbs/git/orb.yml

```
@@ -0,0 +1,72 @@
+---
+version: 2.1
+
+description: |
+  Alternate checkout command to that provided by CircleCI
+
+  CircleCI's `checkout` command has two major issues [[1][1],[2][2]]:
+  - it does shallow clones: you get the latest state of
+    your repository, but don't really get any history;
+  - it messes with branch references;
+
+  Those are not an issue, until your CI workflow has reasons
+  to switch branches or look at the code's history.
+
+  If you do any of those things, you can use this alternate
+  `custom_checkout` command.
+
+  Of course, if CircleCI does things the way it does, is for
```

Just a small wording:
```suggestion
  Of course, if CircleCI does things the way it does, it is for
```